src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Styles/ToggleSwitch.xaml

```diff
@@ -0,0 +1,31 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:wasm="http://uno.ui/wasm"
+                    mc:Ignorable="d wasm">
+
+	<!--  Workaround for #192132 wasm: prefix issue  -->
```

Comment:
Instead of copying this comment. You should open a new issue in uno describe the problem, and link it.
As this is a public repo, other people may not having access to the linked internal work item.
And, maybe describe it as "conditional wasm/skia xmlns doesnt work inside a library".

Reply:
It would beneficial to have mention this in the relevant sample page too.